hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/volume/CreateVolumeHandler.java

```diff
@@ -40,10 +41,15 @@
       description = "Owner of of the volume")
   private String ownerName;
 
+  @Option(names = {"--spaceQuota", "-sq"},
```

Comment:
Do you mind change the "-sq" to "-s" (or other alphabet) ?

I think the command-line promotion is better if it is alignment. 
(And take this way, it would be left-alignment)

![out of alignment](https://i.imgur.com/nXhbsfW.png)
![left alignment](https://i.imgur.com/xzUAiZe.png)